elastic/logs/track.json

```diff
@@ -60,6 +60,12 @@
 {% set p_dsl_default_rollover = (dsl_default_rollover | default(false) ) %}
 {% set p_skip_fleet_globals = (skip_fleet_globals | default(false) ) %}
 
+{% set p_recovery_target = (recovery_target | default('logs-k8-application.log-default') ) %}
```

Comment:
maybe not define default here? Then in `logging-query.json` we can only include `shard-recovery` operation if `recovery_target` track param is defined.

Reply:
I don't think we need to run this on all variants of the track we run in nighly. For example the routing variant doesn't need this.

Reply:
If I don't have a default value the CI would run with an empty value and fail because the index does not exist...this is the reason why I define a default.

Reply:
I am adding a `include_shard_recovery_op` which is `false` by default and that we can eanble only for selected configurations.